index.html

```diff
@@ -197,6 +184,19 @@ <h3 id="resource_wide_defaults">Resource-wide &amp; string-specific defaults</h3
 <p>Low-level support for <a>natural language</a> string metadata is widespread because the use of metadata for storage and interchange of the language of data values is long-established and widely supported in the basic infrastructure of the Web. This includes language attributes in [[XML]] and [[HTML]]; string types in schema languages (e.g. [[xmlschema11-2]]) or the various RDF specifications including [[JSON-LD]]; or protocol- or document format-specific provisions for language.</p>
 
 
+
+<div class="req" id="bp_default_setting">
+<p class="advisement">Specifications MAY define a rule or a field to provide the default language and base direction for all strings in a given resource. However, specifications MUST NOT assume that a resource-wide default is sufficient.  Even if a resource-wide setting is available, it must be possible to use string-specific metadata  to override that default.</p>
```

Comment:
I'm tempted to increase `may` to `should`. I'd suggest fine tuning the first sentence:

> Specifications should define a mechanism, such as specific fields, to provide the default language and the default base direction for all strings in a given resource.